SBTPlugin.html

```diff
@@ -60,6 +60,9 @@
 <h1>SBT Plugin<a class="headerlink" href="#sbt-plugin" title="Permalink to this headline">¶</a></h1>
 <p>Add a line like this to your <cite>project/plugins.sbt</cite> file:</p>
```

Comment:
could you change the copy here to reflect that there's more than one line you need to add?